frontend/src/components/header/SmallHeader.js

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
+// SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+import React, { useState } from "react";
+import {
+	Dialog,
+	DialogContent,
+	DialogTitle,
+	Divider,
+	Slide,
+	Toolbar,
+} from "@mui/material";
+import { Link } from "react-router-dom";
+import NftFullLogo from "../../assets/img/NFTTheWorldFullLogo.svg";
+import RoundedEdgesButton from "../RoundedEdgesButton";
+import { backgroundColor, textColor } from "../../assets/jss/colorPalette";
+import MenuIcon from "@mui/icons-material/Menu";
+import CloseIcon from "@mui/icons-material/Close";
+import HeaderTypography from "../HeaderTypography";
+import LinkActionButton from "../LinkActionButton";
+import Grid from "@mui/material/Grid";
+
+const DialogTransition = React.forwardRef(function Transition(props, ref) {
+	return <Slide direction="down" ref={ref} {...props} >{props.children}</Slide>;
+});
+
+/**
+ * Header that is displayed on mobile devices.
+ * @param userIsAdmin true if current user is an admin
+ * @param user user user object of the currently logged in admin
+ */
+export default function SmallHeader({ userIsAdmin, user }) {
+	const [open, setOpen] = useState(false);
+
+	const toggleMenuDialog = () => setOpen(prevOpen => !prevOpen);
+
+	return <Toolbar style={{ overflowX: "none", display: "flex", justifyContent: "space-between", paddingRight: "6px", paddingLeft: "0px", paddingTop: "3px" }}>
+		<div style={{ display: "inherit", paddingLeft: "4px" }}>
+			<Link to="/" style={{ textDecoration: "none", display: "inherit", userSelect: "none" }}>
+				<img src={NftFullLogo} alt="Logo" style={{ height: "27.55px", marginRight: "7px" }} onDragStart={(e) => e.preventDefault()}/>
```

Comment:
`onDragStart`: is the purpose here to disable what would happen if the image is dragged? Would it prevent dragging transparent copies of the original image to a search bar for example?

Reply:
Yes, that is the exact reason. The user could however still open the dev inspect tools and find the URL to the image there. However, I think for header/footer images it feels more professional this way.